packages/cloudinary/src/index.ts

```diff
@@ -106,21 +105,20 @@ export const outputType: graphql.ObjectType<CloudinaryImage_File> =
     },
   });
 
-export const cloudinaryImage =
-  <ListTypeInfo extends BaseListTypeInfo>({
-    cloudinary,
-    ...config
-  }: CloudinaryImageFieldConfig<ListTypeInfo>): FieldTypeFunc<ListTypeInfo> =>
-  meta => {
+// TODO: no delete support
```

Comment:
This field implicitly preserves the images that are uploaded, as mentioned in https://github.com/keystonejs/keystone/pull/8816#issuecomment-1724786029, we should align this field with the storage approaches we have elsewhere - or at least revisit that design